deploy/charts/cert-manager/README.template.md

```diff
@@ -364,17 +364,24 @@ config:
   kubernetesAPIQPS: 9000
   kubernetesAPIBurst: 9000
   numberOfConcurrentWorkers: 200
+  enableGatewayAPI: true
```

Comment:
This matches @maelvls 's documentation in https://github.com/cert-manager/website/pull/1517
 * https://github.com/cert-manager/website/pull/1517

And I agree that it is useful to add this to the example in the Helm chart README too.

Reply:
mmh I did run `make vendor-go generate-helm-docs` but it did not change any README :thinking:

Reply:
Oh, I think I see what is going on in the Makefile. The template is generated from `values.yaml`. I've made the changes to values.yaml, regenerated the updated template, and repushed it.